supplementary_style_guide/glossary_terms_conventions/general_conventions/k.adoc

```diff
@@ -336,14 +336,27 @@ Do not capitalize the first letter.
 [discrete]
 [[kubelet]]
 ==== image:images/yes.png[yes] kubelet (noun)
-*Description*: In Kubernetes, the kubelet is the agent that controls a Kubernetes node. Each node runs a kubelet, which handles starting and stopping containers on a node, based on the required state defined by the control plane.
+*Description*: In Kubernetes, the _kubelet_ is the agent that controls a Kubernetes node. Each node runs a `kubelet`, which handles starting and stopping containers on a node, based on the required state defined by the master.
 
 *Use it*: yes
 
 *Incorrect forms*: Kubelet
 
 *See also*:
 
+// OCP: General; kept as is
+// TODO: This term is outdated anyway and should be removed in a future update
```

Comment:
Is this comment for the term below? Should we be removing it?

Reply:
Asked a question in the CCS Style Council group chat.

Reply:
Comment needs to be retained.

Reason: Kubernetes might still use 'master' though, but on OCP we replace it with control plane. Entry needs to be revisited at a future date.